bin/db-stg-to-local.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e
+
+echo "*** WARNING: This script will destroy your local DARTS API database and restore it from staging. ***"
+echo "It requires \"az\" \"pg_dump\" and \"psql\", and you must also be connected to the HMCTS VPN and have a postgres database running locally."
+
+command -v az >/dev/null 2>&1 || { echo >&2 "I require \"az\" but it's not installed. Aborting."; exit 1; }
+command -v pg_dump >/dev/null 2>&1 || { echo >&2 "I require \"pg_dump\" but it's not installed. Aborting."; exit 1; }
+command -v psql >/dev/null 2>&1 || { echo >&2 "I require \"psql\" but it's not installed. Aborting."; exit 1; }
```

Comment:
May be worth adding a check for `jq`, as I'm not sure if that comes as standard on our macs.

Reply:
Good shout, thanks.

Reply:
fixed